_includes/skipnav.html

```diff
@@ -1 +1,10 @@
-<a class="usa-skipnav" href="#main-content">Skip to main content</a>
+{% if page.layout == 'page' and page.sidenav %}
+  {% assign has_side_nav = true %}
+{% else %}
+  {% assign has_side_nav = false %}
+{% endif %}
+
+{% if has_side_nav %}
+<a class="usa-skipnav" href="#crt-page--sidenav">Skip to in-page navigation</a>
```

Comment:
I wonder if "Skip to page navigation" might work? Does that sufficiently imply "in-page"?